cmake/templates/setup.bat.in

```diff
@@ -54,8 +54,14 @@ if NOT EXIST %_SETUP_TMP% (
   exit 1
 )
 
-REM invoke Python script to generate necessary exports of environment variables
-%_PYTHON% "%_SETUP_UTIL%" %* > %_SETUP_TMP%
+REM invoke setup script to generate necessary exports of environment variables
+set _SETUP_UTIL_EXE=@SETUP_DIR@/_setup_util.py.exe
```

Comment:
Rather than duplicating the definition I would rather use the existing variable: `set _SETUP_UTIL_EXE=%_SETUP_UTIL%.exe`

Reply:
%_SETUP_UTIL%.exe